pom.xml

```diff
@@ -997,7 +997,7 @@
 
         <bcprov-jdk18.version>1.74.0.wso2v1</bcprov-jdk18.version>
         <bcpkix-jdk18.version>1.74.0.wso2v1</bcpkix-jdk18.version>
-        <org.bouncycastle.imp.pkg.version.range>[1.52.0,2.0.0)</org.bouncycastle.imp.pkg.version.range>
+        <org.bouncycastle.imp.pkg.version.range>[0.0.0,2.0.0)</org.bouncycastle.imp.pkg.version.range>
```

Comment:
All 1.x bc versions are vulnerable.
https://mvnrepository.com/artifact/org.bouncycastle/bc-fips/1.0.2.4

What is the effort of 2.0.0 ?

Reply:
Will track that in a separate effort https://github.com/wso2/product-is/issues/21151